guides/v2.1/cloud/docker/docker-development-debug.md

```diff
@@ -1,112 +1,139 @@
 
-1. Open your PhpStorm project and in the top navigation bar, click ![Start listening for connections]({{ site.baseurl }}/common/images/install_docker_php-storm_xdebug-start-listening.png){:width="25px"}  (**Start listening**)
-1. Put some breakpoints in `pub/index.php` file 
-1. Install debug extension in browser, click **Debug** to enable.
-1. Open `https://localhost` in browser with enabled debug in extension. 
-1. PhpStorm should catch Xdebug connection and you are able to start debugging of web requests.
+1.  Change the Xdebug port to 9001 in the **Languages & Frameworks** > **PHP** > **Debug** > **Xdebug** > **Debug Port** panel.
 
-### Example set up on Chrome {#chrome}
+1.  Click **Apply**.
 
-This section discusses how to use Xdebug in Chrome using the Xdebug Helper extension. For information about Xdebug tools for other browsers, consult the browser documentation.
+## Using Xdebug
 
-To use Xdebug Helper with Chrome:
+The following steps describe debugging web requests and CLI commands.
 
-1.	Install the [Xdebug Helper extension](https://chrome.google.com/webstore/detail/xdebug-helper/eadndfjplgieldjbigjakmdgkmoaaaoc?hl=en) from the Chrome store.
-1.	Enable the extension in Chrome as shown in the following figure.
-	![Enable the Xdebug extension in Chrome]({{ site.baseurl }}/common/images/install_docker_php-storm_xdebug-chrome.png)
-1.	In Chrome, right-click ![Xdebug helper icon]({{ site.baseurl }}/common/images/cloud-xdebug_helper-icon.png){:width="25px"} in the Chrome toolbar.
-1.	From the pop-up menu, click **Options**.
-1.	From the **IDE Key** list, click **PhpStorm**.
-1.	Click **Save**.
-	![Xdebug Helper options]({{ site.baseurl }}/common/images/cloud-xdebug_helper-options.png){:width="400px"}
+#### To debug web requests:
+
+1.  In your PhpStorm project, click ![Start listening for connections]({{ site.baseurl }}/common/images/install_docker_php-storm_xdebug-start-listening.png){:width="25px"} (**Start listening**) in the top navigation bar.
+
+1.  Add breakpoints in the `pub/index.php` file.
+
+1.  Install the debug extension in the browser and click **Debug** to enable.
+
+1.  In the browser, open the `https://localhost` URL with enabled debug in extension. 
+
+1.  When PhpStorm recognizes the Xdebug connection, you can begin debugging web requests.
+
+#### To debug CLI commands:
+
+You can debug any Magento command or PHP script using the following steps.
+
+1.  In your PhpStorm project, open the **Build, Extension, Deployment** > **Docker** panel and click `+` to add a new Docker server and update the following settings:
+
+    -   **Name**: Docker Cloud
+    -   **Connect to Docker daemon with**:
+        - For Windows, select **TCP socket** and update **Engine Api Url** with `tcp://localhost:2375`
+        - For Mac,
```

Comment:
Looks like not finished statement, for Mac it should be by default  `Docker for Mac`